### PR TITLE
feat:completed modal and rating

### PR DESCRIPTION
**Pull request by shashikantmaurya0007**
Screenshot (1604)
Screenshot (1605)

### CODE REVIEW

**Review by manojncsoni007 — APPROVED** [review]
Everything looks good to me

File: Modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+.title_cross_btn {
+    display: flex;
+    justify-content: space-between;
+    padding: 0.3rem 0.5rem;
+}
+
+.modal_desc {
+    padding: 0.3rem 0.5rem;
```
Comment (bishnoimukesh) [inline Modal/modal.css:17]: use px for smaller values

**Comment by pariyar07** [comment]
Liked the UI. Looks good to merge.

File: Rating/rating.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+.rate>input:checked+label:hover,
+.rate>input:checked+label:hover~label,
+.rate>input:checked~label:hover,
+.rate>input:checked~label:hover~label,
+.rate>label:hover~input:checked~label {
+    color: #c59b08;
+}
+
+
+/* Modified from: https://github.com/mukulkant/Star-rating-using-pure-css */
```
Comment (bishnoimukesh) [inline Rating/rating.css:49]: remove this

**Review by bishnoimukesh — APPROVED** [review]
Everything is good
clean UI 
good to merge

**Review by pariyar07 — APPROVED** [review]
Remove some of the commented codes. Liked the UI. Looks good to merge.

**Pull request merged**